README.md

```diff
@@ -172,19 +179,81 @@ The only thing you need to install is Docker. All of the other dev dependencies
     2. All remaining containers can be cleaned up using `iotedgedev docker clean -c`
     3. All remaining images can be cleaned up using `iotedgedev docker clean -i`
 
+### Set up and start modules on a virtual machine
+
+1. [**Set up virtual machine to run Azure IoT Edge**](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-install-iot-edge-ubuntuvm)
+2. **Create and configure an ACR (Azure Container Registry)**
+   1. [Create an Azure Container Registry](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-portal)
+   2. Configure `.env` that was generated in the `iotedgedev init` step with the ACR by modifying these lines with appropriate values:

```

Comment:
>that was generated in the `iotedgedev init` [](http://example.com/codeflow?start=23&length=43)

we should assume that users have not ran through the simulator setup steps. In fact, why don't we swap the order such that VM comes as the first suggested experience and simulator, as an alternative?

Reply:
Actually, I intended for the set up to be  a prereq for both sections, but I failed to nest properly. Do you think it's clear enough now?

Reply:
yes, that looks good to me in the latest iteration.